client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```
@@ -52,11 +51,11 @@
 	}
 
 	.design-selector__option-name {
-		margin-top: 16px;
+		color: var( --studio-gray-40 );
 		display: block;
+		font-size: 14px;
```

[non-blocking] we could use one of our mixins to hopefully keep things maintainable and well aligned in the future 🙏 
```suggestion
		@include onboarding-medium-text;
```